.github/workflows/markdown-lint.yml

```diff
@@ -7,7 +7,7 @@ on:
     paths:
       - '*.md'
```

Comment:
@yin1999 right now this action only runs when a markdown file in the root is touched, not for any of the actual translated files